lib/Sqlite3Client.js

```diff
@@ -0,0 +1,53 @@
+import Client from "./Client.js";
+
+class Sqlite3Client extends Client {
+  getAddNameSql() {
+    return `
+      ALTER TABLE ${this.quotedSchemaTable()} 
+        ADD COLUMN name TEXT;
+    `;
+  }
+
+  getAddMd5Sql() {
+    return `
+      ALTER TABLE ${this.quotedSchemaTable()} 
+        ADD COLUMN md5 TEXT;
+    `;
+  }
+
+  getAddRunAtSql() {
+    return `
+      ALTER TABLE ${this.quotedSchemaTable()} 
+        ADD COLUMN run_at TIMESTAMP WITH TIME ZONE;
+    `;
+  }
+
+  getColumnsSql() {
+    const { config } = this;
+
+    const query = `
+      SELECT name AS column_name
+      FROM pragma_table_info("${config.schemaTable}")
+    `;
+
+    return query;
+  }
+
+  async runQuery(query) {
+    const { config } = this;
+    // Sqlite3 can only run one query at a time
+    const queries = query.split(";");
```

Comment:
This is the oddest bit and possibly the source of problems. We need to return the results of the last query and also support multiple queries at the same time... however sqlite3 would not support that. LMK if you have any better ideas.

Reply:
I have a [package](https://github.com/sqlpad/sql-limiter) that could be leveraged here to split queries if a more advanced means of splitting is necessary! I think this is fine for now though.

Reply:
@rickbergfalk @mcollina Unfortunately some statements don't work because of ';' in the middle, for example:

```sql
CREATE TRIGGER tg_accounts_updated_at                                                                    
AFTER UPDATE ON accounts FOR EACH ROW
BEGIN
  UPDATE accounts
  SET updated_at = CURRENT_TIMESTAMP
  WHERE id = old.id;
END;
```

I tried using `getStatements` from [`sql-limiter`](https://github.com/sqlpad/sql-limiter), but it fails to parse other statements like
```
PRAGMA journal_mode = WAL;
```
or
```
BEGIN TRANSACTION;
```

I'm using the following workaround:
```
-    const queries = query.split(";");
+    const queries = query.split(/(?<!\\);/).map(part => part.replace(/\\;/g, ";"));
```
to allow inner ;-s to be escaped:
```
  WHERE id = old.id\;
END;
```

Reply:
It appears sqlite can execute multiple statements: http://www.sqlite.org/c3ref/exec.html
Some of the wrappers (for example `sqlite`) as well as experimental native NodeJS integration exports it: https://nodejs.org/api/sqlite.html
so using `exec` is probably better approach.

Reply:
On further check, `exec` is supported by `sqlite3` npm module which is being used by postgresor / postgresor-cli.
But using it would require removing splitting by `;` here, and doing something like this in config.execQuery: `if query.contains(';') { db.exec() } else { db.all() }` OR splitting config.execQuery into config.execQuery and config.execSqlScript (for user script with multiple statements). The latter can fallback to execQuery by default if not specified. What do you think?

Reply:
@zmeyc thinking about this and I'm leaning towards another function to execute the migration scripts like `config.execSqlScript` as you mention, and falling back if not provided. It seems like it'd be less problematic in the long run. Trying to think of other scenarios. 

I'm not sure I have a strong opinion either way though.

@mcollina any thoughts on this?

Reply:
Opened an issue so we don't lose track of this :sweat_smile: https://github.com/rickbergfalk/postgrator/issues/166

Reply:
using the "split" approach would not really work as there are a few edge cases that are impossible to handle without parsing the full SQL grammar.

Adding an additional config would be best.